lib/scroll-through-time.js

```diff
@@ -2,16 +2,37 @@
 
 import { CompositeDisposable } from 'atom';
 
-function getListener (direction) {
-  let mainAxis = direction == 'left' || direction == 'right' ? 'X' : 'Y'
+function getListener (options) {
+  let mainAxis = options.direction == 'left' || options.direction == 'right' ? 'X' : 'Y'
   let crossAxis = mainAxis == 'X' ? 'Y' : 'X'
-  let reversed = direction == 'right' || direction == 'down'
+  let reversed = options.direction == 'right' || options.direction == 'down'
+  let keyboard = options.keyboard
   let previousScroll = 0
   let timeout
   return function (e) {
-    if (Math.abs(e['delta' + mainAxis]) < Math.abs(e['delta' + crossAxis])) {
-      return
+
+      function isDescendantOfType(parentType, child) {
```

Comment:
define the function on the root scope so that it doesn't need to be defined everytime

Reply:
Can do!